test_identity.schema.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This file should be placed in the `ory_mocking` folder.

Reply:
done